postgres/initdb.sql

```diff
@@ -4,7 +4,7 @@ CREATE EXTENSION postgis;
 CREATE TABLE IF NOT EXISTS projects (
     created timestamp,
     created_by varchar,
-    geom geometry(MULTIPOLYGON, 4326),
+    geom geometry,
```

Comment:
okay since this will have an effect on the database tables which are already in place, we need to add a script to the postgres folder to alter the geom column for projects and